src/nvim/ops.c

```diff
@@ -2927,13 +2926,14 @@ void do_put(int regname, yankreg_T *reg, int dir, long count, int flags)
      */
     if (y_type == MCHAR && y_size == 1) {
       do {
-        totlen = count * yanklen;
+        assert(count <= INT_MAX);
```

Comment:
a more relevant assert would be `count >= 0` but as said not 100% sure that would be correct. I suppose we could try it and see if it trigger on travis.